website/content/api-docs/secret/ldap.mdx

```diff
@@ -181,6 +184,10 @@ The `static-role` endpoint configures Vault to manage the passwords of existing
 - `rotation_period` `(string: <required>)` - How often Vault should rotate the password of the user entry. Accepts
   [duration format strings](/vault/docs/concepts/duration-format). The minimum rotation period is 5 seconds.<br />
   **Example:** `"3600", "5s", "1h"`
+- `skip_import_rotation` `(boolean: false)` - When set on a role creation request, Vault will not rotate the
```

Comment:
I don't think it matters but I noticed we are using "bool" and "boolean". We are not consistent in our docs across vault but it looks like we mostly use "bool"